### PR TITLE
Feature/cp 25/player and branding update

### PR DESCRIPTION
**Pull request by bengo1990**
Appropriate TODOs placed within the code for phase 2 of this PR (when doc locations get finalised, etc).

package.json file will also need amending in the future... can't put a TODO in the file.

### CODE REVIEW

File: src/studiodrm-dashjs.js [hunk 1]
```diff
@@ -76,27 +77,31 @@
...
     var playReadyLaUrl =
-        "https://playready-license.drm.technology/rightsmanager.asmx?token=" +
-        encodeURIComponent(vudrmToken);
+        "https://playready-license.drm.technology/rightsmanager.asmx";
     // For Widevine set the LaURL and the VUDRM token.
     var widevineLaUrl = "https://widevine-proxy.drm.technology/proxy";
 
-    // Set the protection data. dash.js only supports PlayReady and Widevine but Vualto do support!
+    // TODO: Change lines 97, 103 with the new header name... X-STUDIODRM-TOKEN?
+    // Set the protection data. dash.js only supports PlayReady and Widevine but JW Player do support!
```
Comment (bengo1990) [inline src/studiodrm-dashjs.js:92]: "dash.js only supports PlayReady and Widevine but JW Player do support!"

what is this meant to say? :)

File: README.md [hunk 2]
```diff
@@ -1,48 +1,52 @@
...
+﻿# Studio DRM dash.js
 
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 [![Built with Grunt](http://cdn.gruntjs.com/builtwith.svg)](https://gruntjs.com/)
 
 ## Description
 
-This repository will demonstrate how to use [VUDRM](https://vudrm.vualto.com/) with the [dash.js](https://github.com/Dash-Industry-Forum/dash.js/wiki).
-If you have any questions please contact support@vualto.com
+#### TODO: Change the below to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:8]: TODOs need the jira link

File: README.md [hunk 3]
```diff
@@ -1,48 +1,52 @@
...
-This repository is currently targeted at [version 3.0.0](https://github.com/Dash-Industry-Forum/dash.js/releases/tag/v3.0.0) of dash.js.
+This repository is currently targeted at [version 4.3.0](https://github.com/Dash-Industry-Forum/dash.js/releases/tag/v4.3.0) of dash.js.
 
 ## Instructions
 
 ### Install dependencies
 
 1. Install [npm](https://www.npmjs.com/)
 2. Install the [grunt-cli](https://www.npmjs.com/package/grunt-cli): `npm install -g grunt-cli`
+#### TODO: Change the below (no.3) to the JW GitHub repo structure name
```
Comment (somerobmitchell) [inline README.md:20]: TODOs need the jira link

File: README.md [hunk 4]
```diff
@@ -1,48 +1,52 @@
...
 
 ### Build and run the dev environment
 
 1. Open the repository in your favourite javascript editor.
-2. In file `src/vuplay-dashjs.js` replace `<your-stream-url>` with your stream URL. This must be a [MPEG-DASH](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) stream.
-3. In file `src/vuplay-dashjs.js` replace `<your-vudrm-token>` with a VUDRM token from [https://admin.drm.technology](https://admin.drm.technology)
-4. Run `npm run build` in the project's root. This will create a `dist` folder that contains all the files needed to run this demo. N.B. You will need to add the host `dashjs.vuplay.co.uk` to your local machine's hosts file in order for this to work.
-5. Load a supported browser and go to `https://dashjs.vuplay.co.uk:14703`
+2. In file `src/studiodrm-dashjs.js` replace `<your-stream-url>` with your stream URL. This must be a [MPEG-DASH](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) stream.
+#### TODO: Change the below to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:29]: TODOs need the jira link

File: README.md [hunk 5]
```diff
@@ -1,48 +1,52 @@
...
 ## Useful links
 
-### VUDRM
+### Studio DRM
 
--   [Contact vualto](https://www.vualto.com/contact-us/)
--   [VUDRM](https://vudrm.vualto.com/)
--   [VUDRM token documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html)
+-   [Contact JW Player](https://support.jwplayer.com/)
+#### TODO: Change the below to the JW Studio DRM docs
```
Comment (somerobmitchell) [inline README.md:47]: TODOs need the jira link

File: src/studiodrm-dashjs.js [hunk 6]
```diff
@@ -76,27 +77,31 @@
...
     // For PlayReady the VUDRM token is attached as a querystring parameter on the license server URL.
     var playReadyLaUrl =
-        "https://playready-license.drm.technology/rightsmanager.asmx?token=" +
-        encodeURIComponent(vudrmToken);
+        "https://playready-license.drm.technology/rightsmanager.asmx";
     // For Widevine set the LaURL and the VUDRM token.
     var widevineLaUrl = "https://widevine-proxy.drm.technology/proxy";
 
-    // Set the protection data. dash.js only supports PlayReady and Widevine but Vualto do support!
+    // TODO: Change lines 97, 103 with the new header name... X-STUDIODRM-TOKEN?
```
Comment (somerobmitchell) [inline src/studiodrm-dashjs.js:91]: TODOs need the jira link

File: src/studiodrm-dashjs.js (hunk 1 above)
Comment (somerobmitchell) [reply src/studiodrm-dashjs.js:92]: I think just remove the "but...." bit on the end.

**Review by somerobmitchell — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (bengo1990) [reply README.md:47]: Actioned.

File: README.md (hunk 3 above)
Comment (bengo1990) [reply README.md:20]: Actioned.

File: README.md (hunk 2 above)
Comment (bengo1990) [reply README.md:8]: Actioned.

File: src/studiodrm-dashjs.js (hunk 1 above)
Comment (bengo1990) [reply src/studiodrm-dashjs.js:92]: Actioned.

**Comment by bengo1990** [comment]
Spoke to Kevin - he has asked to wait for him to finish the Token v2 JW docs prior to this being merged in. Please review the document, if there are no changes, I will merge when he is happy.

**Review by somerobmitchell — DISMISSED** [review]

File: src/studiodrm-dashjs.js [hunk 7]
```diff
@@ -76,27 +76,30 @@
...
         document.querySelector("#ttml-rendering-div"),
     );
 
-    // For PlayReady the VUDRM token is attached as a querystring parameter on the license server URL.
+    // For PlayReady the Studio DRM token is attached as a querystring parameter on the license server URL.
     var playReadyLaUrl =
-        "https://playready-license.drm.technology/rightsmanager.asmx?token=" +
-        encodeURIComponent(vudrmToken);
-    // For Widevine set the LaURL and the VUDRM token.
+        "https://playready-license.drm.technology/rightsmanager.asmx";
```
Comment (PerritonBranch) [inline src/studiodrm-dashjs.js:86]:
```suggestion
        "https://playready-license.vudrm.tech/rightsmanager.asmx";
```

File: src/studiodrm-dashjs.js [hunk 8]
```diff
@@ -76,27 +76,30 @@
...
+    // For Widevine set the LaURL and the Studio DRM token.
     var widevineLaUrl = "https://widevine-proxy.drm.technology/proxy";
```
Comment (PerritonBranch) [inline src/studiodrm-dashjs.js:88]:
```suggestion
    var widevineLaUrl = "https://widevine-proxy.vudrm.tech/proxy";
```

**Review by PerritonBranch — DISMISSED** [review]

File: src/studiodrm-dashjs.js (hunk 7 above)
Comment (bengo1990) [reply src/studiodrm-dashjs.js:86]: Actioned.

File: src/studiodrm-dashjs.js (hunk 8 above)
Comment (bengo1990) [reply src/studiodrm-dashjs.js:88]: Actioned.

**Review by PerritonBranch — APPROVED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**